server/src/main.rs

```diff
@@ -52,6 +53,11 @@ async fn main() -> std::io::Result<()> {
             .app_data(web::JsonConfig::default().limit(MAX_JSON_PAYLOAD))
             .service(source_code_handler)
             .service(health_handler)
+            .service(
+                web::scope("/api/calendar")
+                    .configure(calendar::configure)
+                    .app_data(last_sync.clone()),
+            )
```

Comment:
This shouldn't be part of the 'server' main anymore, or am I misunderstanding?

Reply:
Good catch!